features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -1032,6 +1032,10 @@
         <!--<EnableErrorCodeForPasswordPolicyViolation>{{scim2.enable_error_code_for_password_policy_violations}}</EnableErrorCodeForPasswordPolicyViolation>-->
     </SCIM2>
 
+    <PasswordPolicy>
+        <MaxPasswordAllowedLength>64</MaxPasswordAllowedLength>
```

Comment:
What is the reasoning behind choosing 64 as the default value?

Reply:
We want to increase the current number 30 and I had a discussion with ashen and omindu regarding this. In there we have decided to increase the number to 64.

Reply:
Ack